lesson_07/homework_07.py

```diff
@@ -0,0 +1,182 @@
+import re
+
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+
+def delimiter(num):
+    print()
+    print(f"Task {num}")
+
+delimiter(1)
+
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+    if number >= 1:
+        #Complete the while loop condition.
+        while multiplier <= 25:
+            result = number * multiplier
+            if  result > 25 or result == 0:
+                break
+            print(number, "x", multiplier, "=", result)
+
+            # Increment the appropriate variable
+            multiplier += 1
+    else: print("Not valid number")
```

Comment:
```suggestion
    else: 
        print("Not valid number")
```
it is easier to read if you write it on the next line